openpype/modules/kitsu/utils/update_op_with_zou.py

```diff
@@ -33,16 +39,29 @@ def create_op_asset(gazu_entity: dict) -> dict:
     }
 
 
+def get_kitsu_project_name(project_id: str):
```

Comment:
```suggestion
def get_kitsu_project_name(project_id: str)->str:
```

Reply:
I think this should belong into an `utils.py` file as it's shared into both `update_` python files.